style.css

```diff
@@ -0,0 +1,105 @@
+*{
+  padding: 0;
+  margin: 0;
+}
```

Comment:
I think you should add a `box-sizing: border-box. This will help manage the extra padding and borders, thereby enabling you to keep track of element width. 👍